release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1833,6 +1833,11 @@ This issue affects CPU load-balancing features, which depend on the CPU Manager
 
 * When a pod uses the CNI plugin for DHCP address assignment in conjunction with other CNI plugins, the network interface for the pod might be unexpectedly deleted. As a result, when the DHCP lease for the pod expires, the DHCP proxy enters a loop when trying to re-create a new lease, leading to the node becoming unresponsive. There is currently no workaround. (link:https://issues.redhat.com/browse/OCPBUGS-45272[*OCPBUGS-45272*])
 
+[id="ocp-nodes-4-18-known-issues_{context}"]
+
+* When using PXE boot to xref:../nodes/nodes/nodes-nodes-adding-node-iso.adoc#adding-node-iso[add a worker node to an on-premise cluster], sometimes the host fails to reboot from the disk properly, preventing the installation from completing.
```

Comment:
By right, worker node should be compute node, but the target doc says "worker node". I'll leave it to you as what is suitable in this situation.